studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -1,106 +1,140 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { groupBy } from 'lodash'
+import { observer } from 'mobx-react-lite'
 import Link from 'next/link'
-import { FC } from 'react'
 import { Badge, Button, IconPlus } from 'ui'
-import { observer } from 'mobx-react-lite'
 
+import {
+  OverdueInvoicesResponse,
+  useOverdueInvoicesQuery,
+} from 'data/invoices/invoices-overdue-query'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { usePermissionsQuery } from 'data/permissions/permissions-query'
+import { useProjectsQuery } from 'data/projects/projects-query'
+import { useCheckPermissions } from 'hooks'
 import { IS_PLATFORM } from 'lib/constants'
-import { checkPermissions, useStore } from 'hooks'
-import { Organization, Project } from 'types'
 import { makeRandomString } from 'lib/helpers'
+import { Organization, Project } from 'types'
 import ProjectCard from './ProjectCard'
 import ShimmeringCard from './ShimmeringCard'
-import { PermissionAction } from '@supabase/shared-types/out/constants'
-import { useOverdueInvoicesQuery } from 'data/invoices/invoices-overdue-query'
 
-interface Props {
+export interface ProjectListProps {
   rewriteHref?: (projectRef: string) => string
 }
 
-const ProjectList: FC<Props> = ({ rewriteHref }) => {
-  const { app, ui } = useStore()
-  const { organizations, projects } = app
-  const { isLoading: isLoadingProjects } = projects
-
+const ProjectList = ({ rewriteHref }: ProjectListProps) => {
+  const { data: organizations } = useOrganizationsQuery()
+  const { data: allProjects, isLoading: isLoadingProjects } = useProjectsQuery()
+  const { isLoading: _isLoadingPermissions } = usePermissionsQuery()
```

Comment:
nit: do we need `_` here?

Reply:
Happy to change it if you can think of a better name. It's because a few lines down I'm making another variable called: `isLoadingPermissions`